sos/report/plugins/ceph_iscsi.py

```diff
@@ -0,0 +1,36 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, RedHatPlugin, UbuntuPlugin
+
+
+class CephISCSI(Plugin, RedHatPlugin, UbuntuPlugin):
+
+    short_desc = "CEPH iSCSI"
+
+    plugin_name = "ceph_iscsi"
+    profiles = ("storage", "virt", "container")
+    packages = ("ceph-iscsi",)
+    services = ("rbd-target-api", "rbd-target-gw")
+    containers = ("rbd-target-api.*", "rbd-target-gw.*")
+
+    def setup(self):
+        self.add_copy_spec([
+            "/etc/tcmu/tcmu.conf",
+            "/var/log/ceph/ceph-client.iscsi.log",
```

Comment:
Cycling back around on this. I think we just need this to be `/var/log/ceph/**/ceph-client.iscsi.log` and this one will be good to merge.